### DISCUSSION TITLE
My External Apple 27" Thunderbolt Display Went Dim

### DISCUSSION THREAD

**Discussion by tonitrillionaire** [MonitorControl/MonitorControl#1324] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Please help! I just installed MonitorControl. I have no idea how I made my Apple 27" Display Monitor dim. I should not have downloaded the advanced MonitorControl app yet here I am. I've don't speak developer lingo, so can you make it simple as possible? I clicked one of those bottom two options and it went dim.

I clicked on the **disable macos OSD** and the **avoid gamma table manipulation** option and my monitor went dim, not sure which option made it dim.
<img width="731" alt="Screenshot 2023-02-27 at 8 25 54 PM" src="https://user-images.githubusercontent.com/126531223/221729385-6432224b-7392-4c74-ac02-2ffb1638b5f1.png">

### Expected behavior

I expected my monitor to get brighter. I see it based on the ambient light, if it's dark it gets darker, if it's light it is fine. How do I turn off that feature?

### Environment Information (please complete the following information)

```markdown
- macOS version: Ventura 13.0.1
- Mac model: Macbook Pro 16-inch, 2019
- MonitorControl version: 4.1.0
- Monitor(s): Apple 27" Display Monitor
- Monitor Cable(s)/Connection(s): Thunderbolt cable with USBC adapter 
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
Hi there, it is probably because these displays might use a non-standard 0-255 DDC brightness range. You can change the range by enabling `Show advanced settings`. An other route you might want to try is to issue a DDC factory reset that should reset the display to its default settings (I am not sure if this works with the 27" Thunderbolt Display) - to do this, you can download [BetterDisplay](https://github.com/waydabber/BetterDisplay#readme) and from the app menu (after you expand with the chevron) select DDC Control / DDC Factory Reset. You can also set the range to 0-255 easily in BetterDisplay under DDC Control / DDC Sliders Quick Config.

Hope this helps!
